docs/install/ide.mdx

```diff
@@ -37,14 +41,8 @@ Features:
 
 To install, go to the IDE's plugin browser and search for `Bazel`.
 
-To manually install older versions, download the zip files from JetBrains'
-Plugin Repository and install the zip file from the IDE's plugin browser:
-
-*  [Android Studio
-   plugin](https://plugins.jetbrains.com/plugin/9185-android-studio-with-bazel)
-*  [IntelliJ
-   plugin](https://plugins.jetbrains.com/plugin/8609-intellij-with-bazel)
-*  [CLion plugin](https://plugins.jetbrains.com/plugin/9554-clion-with-bazel)
+To manually install older versions, download the zip files from the JetBrains
+Marketplace and install the zip file from the IDE's plugin browser.
```

Comment:
We also publish builds on GitHub for CLwB (and legacy IJwB): https://github.com/bazelbuild/intellij/releases